README.md

```diff
@@ -19,6 +22,19 @@ npm install -D jest jest-preset-angular @types/jest
 
 This will install `jest`, `@types/jest`, `ts-jest` as dependencies needed to run with Angular projects.
 
+Additionally you need `babel` packages if you want to use `babel`.
+
+### babel
+`babel` uses the babel compiler to generate JavaScript, without type-checking your test files before doing so. This might result in a different test run performance. You can still run type-checking using `tsc --noEmit`. For additional TypeScript-supported language features you might have to install more babel packages. Note that `babel` also can differ slightly from `tsc`, e. g. in compiling a class to a function.
```

Comment:
```suggestion
`babel` preset uses the Babel compiler to generate JavaScript without type-checking your test files before doing so. This might result in a different test run performance. You can still run type-checking using `tsc --noEmit`. For additional TypeScript-supported language features you might have to install more babel packages. Note that `babel` also can differ slightly from `tsc`, e.g. in compiling a class to a function.
```